contributing/community/mentoring.rst

```diff
@@ -0,0 +1,12 @@
+Mentoring
+=========
+
+Reading the :doc:`contributing </contributing/index>` is already a great way
+to get started on becoming a Symfony contributor. However, sometimes
+it might still seem overwhelming and for this purpose we created a
+dedicated slack channel called `contribs`_ to connect new contributors
```

Comment:
Slack (with an uppercase)

Reply:
Can/should we link `Slack Channel` to the place where someone can sign up for the Slack? In case this is their first time wanting to do that?